test/k8sT/manifests/1.7/cilium_ds_geneve.yaml

```diff
@@ -0,0 +1,187 @@
+  namespace: kube-system
+- kind: Group
+  name: system:nodes
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: cilium
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: cilium
+        kubernetes.io/cluster-service: "true"
+    spec:
+      containers:
+      - image: 192.168.36.11:5000/cilium/cilium-dev:latest
+        imagePullPolicy: Always
+        name: cilium-agent
+        command: [ "cilium-agent" ]
+        args:
+          - "--debug"
+          - "--kvstore"
+          - "etcd"
+          - "--kvstore-opt"
+          - "etcd.address=http://192.168.36.11:9732"
+          - "--k8s-kubeconfig-path"
+          - "/opt/cilium/kubeconfig"
+          - "-t"
+          - "geneve"
+        lifecycle:
+          postStart:
+            exec:
+              command:
+                - "/cni-install.sh"
+          preStop:
+            exec:
+              command:
+                - "/cni-uninstall.sh"
+        env:
+          - name: "K8S_NODE_NAME"
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+        livenessProbe:
+          exec:
+            command:
+            - cilium
+            - status
+          initialDelaySeconds: 30
+          failureThreshold: 10
+          periodSeconds: 10
+        readinessProbe:
+          exec:
+            command:
+            - cilium
+            - status
+          initialDelaySeconds: 30
+          periodSeconds: 15
+        volumeMounts:
+          - name: bpf-maps
+            mountPath: /sys/fs/bpf
+          - name: cilium-run
+            mountPath: /var/run/cilium
+          - name: cni-path
+            mountPath: /host/opt/cni/bin
+          - name: etc-cni-netd
+            mountPath: /host/etc/cni/net.d
+          - name: docker-socket
+            mountPath: /var/run/docker.sock
+            readOnly: true
+          - name: kubeconfig-path
+            mountPath: /opt/cilium/
+            readOnly: true
+        securityContext:
+          capabilities:
+            add:
+              - "NET_ADMIN"
+          privileged: true
+      hostNetwork: true
+      volumes:
+        - name: cilium-run
+          hostPath:
+            path: /var/run/cilium
+        - name: cni-path
+          hostPath:
+            path: /opt/cni/bin
+        - name: bpf-maps
+          hostPath:
+            path: /sys/fs/bpf
+        - name: docker-socket
+          hostPath:
+            path: /var/run/docker.sock
+        - name: etc-cni-netd
+          hostPath:
+              path: /etc/cni/net.d
+        - name: kubeconfig
+          hostPath:
+              path: /etc/kubernetes/kubelet.conf
+        - name: kubeconfig-path
+          hostPath:
+              path: /opt/cilium/
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+      - effect: NoSchedule
+        key: node.cloudprovider.kubernetes.io/uninitialized
+        value: "true"
```

Comment:
Having this many DaemonSet files to maintain will be very difficult for us, as we have to update `examples/minikube`, `examples/kubernetes`, etc. Is there a way we can just use the files located in `examples/` directory instead of duplicating them here?